slices/procps.yaml

```diff
@@ -4,6 +4,11 @@ essential:
   - procps_copyright
 
 slices:
+  standard:
```

Comment:
see my comment in the review, but I think this `standard` might actually be a `core`?